tools/osv-scanner/cmd/osv-scanner/main.go

```diff
@@ -19,14 +19,20 @@ import (
 )
 
 // scanDir walks through the given directory to try to find any relevant files
-func scanDir(query *osv.BatchedQuery, dir string, skipGit bool) error {
+func scanDir(query *osv.BatchedQuery, dir string, skipGit bool, recursive bool) error {
 	log.Printf("Scanning dir %s\n", dir)
+	root := true
 	return filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
 		if err != nil {
 			log.Printf("Failed to walk %s: %v", path, err)
 			return err
 		}
 
+		if !root && !recursive && info.IsDir() {
```

Comment:
This will skip the .git scanning on the next if block. Can you move this to the bottom of the function instead?